src/util/Util.js

```diff
@@ -573,6 +573,34 @@ class Util {
       setTimeout(resolve, ms);
     });
   }
+  
+  /**
+   * Checks whether there is a custom emoji in the message content
+   * @param {string} text to check
+   * @returns {booelan}
+   */
+  static includesCustomEmoji(text) {
+        if (text !== String) throw new TypeError('Value is not a string')
```

Comment:
constructing this error will result in throwing an error, as `TypeError` is defined as the custom djs error, see [here](https://github.com/discordjs/discord.js/blob/161f90761aa357f17066f4171a5f43f76a552eb7/src/util/Util.js#L5), [here](https://github.com/discordjs/discord.js/blob/161f90761aa357f17066f4171a5f43f76a552eb7/src/errors/DJSError.js#L59), [and here](https://github.com/discordjs/discord.js/blob/161f90761aa357f17066f4171a5f43f76a552eb7/src/errors/DJSError.js#L40), and you haven't provided a valid error message key, consider using the INVALID_TYPE error message key